src/main/java/it/gov/pagopa/pu/workflow/wf/receiptclassification/iuf/dto/IufReceiptClassificationForReportingSignalDTO.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.dto;
+
+
+import it.gov.pagopa.payhub.activities.dto.classifications.Transfer2ClassifyDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@Builder
+public class IufReceiptClassificationForReportingSignalDTO {
```

Comment:
just call it `IufClassification`
```suggestion
public class IufClassificationPaymentsReportingSignalDTO {
```

Reply:
IufClassificationNotifyPaymentsReportingSignalDTO